app/views/contexts/summary/answer.py

```diff
@@ -1,18 +1,22 @@
+from typing import Any, Mapping, Optional
+
 from flask import url_for
 
+from app.data_models.answer import AnswerValueEscapedTypes
+
 
 class Answer:
     def __init__(
         self,
         *,
-        answer_schema,
-        answer_value,
-        block_id,
-        list_name,
-        list_item_id,
-        return_to,
-        return_to_block_id,
-    ):
+        answer_schema: Mapping[str, Any],
+        answer_value: Optional[dict[str, Optional[AnswerValueEscapedTypes]]],
```

Comment:
This isn't always a dict, it should be `Union[dict, None, AnswerValueEscapedTypes]`?

Reply:
Sorry its actually `Union[None, dict[str, Optional[AnswerValueEscapedTypes]], str, AnswerValueEscapedTypes]`